etc/bin/swarm

```diff
@@ -204,7 +204,7 @@ append_config () {
         if prompt "Write \"$LINE\" to \"$LOC\"?"; then
             if [[ "$POS" == top ]]; then
                 # ... at the top of the file
-                echo -e "${LINE}\n$(cat ${LOC})" > "$LOC"
+                echo -e "${LINE}\n$(cat "$LOC")" > temp && mv temp "$LOC"
```

Comment:
```suggestion
                echo -e "${LINE}\n$(cat "$LOC")" > "$LOC"
```

Reply:
@MetRonnie I'm gonna merge this without the temp file, to stop other CI runs failing on the shellcheck. You can always propose another one-liner if you disagree!